appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -135,8 +137,13 @@ class LoggedInFlowNode @AssistedInject constructor(
         }
     }
 
+    private val navigateBackToRoomObserverPlugin = NavigateBackToRoomObserverPlugin()
+
     override fun onBuilt() {
         super.onBuilt()
+
+        Appyx.globalPlugins = listOf(navigateBackToRoomObserverPlugin)
```

Comment:
Maybe cleaner to do
```
Appyx.globalPlugins += navigateBackToRoomObserverPlugin
```
in the onCreate block below 
and remove the plugin in the `onDestroy` with
```
Appyx.globalPlugins -= navigateBackToRoomObserverPlugin
```

Reply:
`Appyx.globalPlugins += navigateBackToRoomObserverPlugin` was compiling when I tested locally (to be more precise: the IDE did not complain), but yes, it's worth checking again that this is still working as expected

Reply:
Oh, I think you're right. I guess it's creating a new list with the previous contents, since the signature is `val globalPlugins: List<Plugin>`.

Reply:
Yes it should word, because the globalPlugins is a var, so it'll copy the value and add the new one.
But I find it weird to use a globalPlugin for that...

Reply:
It's either that or passing down the plugin until it reaches the required flow node, and that has bitten us in the past. I can do it like that though, if you think it's best.